integration/playwright/.env.example

```
@@ -0,0 +1,13 @@
+GRAZ_E2E_WALLET_MNEMONIC=
+GRAZ_E2E_CHAIN_ID=cosmoshub-4
+GRAZ_E2E_CHAIN_NAME=Cosmos Hub
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAZ_E2E_CHAIN_NAME key should go before the GRAZ_E2E_WALLET_MNEMONIC key

(UnorderedKey)